packages/js/e2e-utils-playwright/src/plugin-utils.js

```diff
@@ -0,0 +1,179 @@
+const { APIRequest } = require( '@playwright/test' );
+const axios = require( 'axios' ).default;
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { promisify } = require( 'util' );
+const execAsync = promisify( require( 'child_process' ).exec );
+
+/**
+ * Encode basic auth username and password to be used in HTTP Authorization header.
+ *
+ * @param {string} username
+ * @param {string} password
+ * @returns Base64-encoded string
+ */
+export const encodeCredentials = ( username, password ) => {
+	return Buffer.from( `${ username }:${ password }` ).toString( 'base64' );
+};
+
+/**
+ * Download the zip file from a remote location.
+ *
+ * @param {object} param
+ * @param {string} param.url
+ * @param {string} param.repository
+ * @param {string} param.authorizationToken
+ * @param {boolean} param.prerelease
+ * @param {string} param.downloadDir
+ *
+ * @param {string} url The URL where the zip file is located. Takes precedence over `repository`.
+ * @param {string} repository The repository owner and name. For example: `woocommerce/woocommerce`. Ignored when `url` was given.
+ * @param {string} authorizationToken Authorization token used to authenticate with the GitHub API if required.
+ * @param {boolean} prerelease Flag on whether to get a prelease or not. Default `false`.
+ * @param {string} downloadDir Relative path to the download directory. Non-existing folders will be auto-created. Defaults to `tmp` under current working directory.
+ *
+ * @return {string} Absolute path to the downloaded zip.
+ */
+export const downloadZip = async ( {
```

Comment:
I don't know that this utility is in use anywhere anymore.  It might be leftover from days when we were installing updated versions of WC on remote web hosts.

Reply:
Regardless, I don't think it belongs in a general "Playwright test utilities" package.  I think we can remove it altogether.